lib/theme.dart

```diff
@@ -4,59 +4,75 @@ import 'package:flutter/material.dart';
 // Primary Color
 // https://maketintsandshades.com/#048ABF
 const _onPrimaryColorValue = 0xFF048ABF;
-const _onPrimaryColorMaterial = MaterialColor(
+const onPrimaryColorMaterial = MaterialColor(
   _onPrimaryColorValue,
   <int, Color>{
-    50: Color(0xFFE6F3F9), // Tint 90%
-    100: Color(0xFFCDE8F2), // Tint 80%
-    200: Color(0xFF9DB0E5), // Tint 60%
-    300: Color(0xFF68B9D9), // Tint 40%
-    400: Color(0xFF36A1CC), // Tint 20%
-    500: Color(_onPrimaryColorValue),
-    600: Color(0xFF036E99), // Shade 20%
-    700: Color(0xFF025373), // Shade 40%
-    800: Color(0xFF02375C), // Shade 60%
-    900: Color(0xFF011C26) // Shade 80%
+    50: Color(0xFFF2FBFC),
+    100: Color(0xFFC0EBEF),
+    200: Color(0xFF81D6DF),
+    300: Color(0xFF04ADC0),
+    350: Color(0xFD048ABF), //30% transparent
```

Comment:
I thought we weren't doing transparency in our materials.

Reply:
This is the only one because we have a transparent color in Figma right now. We might get rid of this in the future idk